c_binding/libsg.h

```diff
@@ -79,6 +79,12 @@
 #define _SG_T10_SPC_MODE_SENSE_MAX_LEN              0xffff
 /* ^ SPC-5 rev 12, 6.15 MODE SENSE(10) command introduction */
 
+#define _T10_SPC_LOG_SENSE_MAX_LEN                  0xffff
+/* ^ SPC-5 rev07 6.8 LOG SENSE command */
+
+#define _T10_SPC_REQUEST_SENSE_MAX_LEN              0xffff
```

Comment:
@tasleson I don't disagree, but it seemed like precedent had been to set that command size for other commands that wouldn't theoretically require that much space either. I can change it and test to make sure it works.